vmdb/app/models/vmdb_database_connection.rb

```diff
@@ -17,27 +29,67 @@ class VmdbDatabaseConnection < ActsAsArModel
   virtual_belongs_to :miq_worker
 
   virtual_column :pid, :type => :integer
+  virtual_column :blocked_by, :type => :integer
+
+  attr_reader :vmdb_database_id
+
+  def vmdb_database_id
+    @vmdb_database_id ||= self.class.vmdb_database.id
+  end
 
-  def initialize(values = {})
-    values[:vmdb_database] ||= self.class.vmdb_database
-    super(values)
+  def address
+    client_addr
   end
```

Comment:
@tenderlove I read that as Rails is :sparkles:(magic) sometimes and even you can't remember what those methods do...  Glad to hear... I agree, keep it simple and obvious.